packages/fscomponents/src/components/CategoryLine.tsx

```diff
@@ -19,19 +19,28 @@ export interface CategoryLineProps extends CommerceTypes.Category {
   style?: StyleProp<ViewStyle>;
   titleStyle?: StyleProp<TextStyle>;
   underlayColor?: string;
+  imageBool?: boolean;
```

Comment:
Can we rename these to something more descriptive? Like `shouldShowImage` or just `showImage`?